.github/workflows/test-coverage.yml

```
@@ -9,19 +9,28 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    env:
+      # Ensure directory-based coverage is not enabled; avoids requiring `go tool covdata`
+      GOCOVERDIR: ""
     steps:
       - name: Checkout Code
         uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)